src/WallsComputation.cpp

```diff
@@ -125,10 +127,17 @@ void WallsComputation::generateInsets(SliceLayerPart* part)
         }
     }
 
+    // When spiralizing, generate the spiral insets using simple offsets instead of generating toolpaths
+    if (spiralize)
+    {
+        generateSpiralInsets(part, line_width_0, wall_0_inset, recompute_outline_based_on_outer_wall);
+    }
     // Call on libArachne:
     WallToolPaths wall_tool_paths(part->outline, line_width_0, line_width_x, inset_count, settings);
     part->wall_toolpaths = wall_tool_paths.getToolPaths();
     part->inner_area = wall_tool_paths.getInnerContour();
+    part->print_outline = part->outline;
+    // TODO Recompute the print_outline of the part according to the generated wall_toolpaths
```

Comment:
Oh, but this could actually be a pretty big problem, right?

Reply:
Generally, yes, it won't be easy to implement. Nevertheless, we can just keep it as the part->outline offsetted by the outer wall inset, as the deviation between the outer side of the outer wall generated by libArachne and the print outline should be insignificant.